api/structure/schema.py

```diff
@@ -0,0 +1,48 @@
+import graphene
+from graphene_sqlalchemy import SQLAlchemyObjectType
+
+from api.structure.models import Structure as StructureModel
+from helpers.auth.authentication import Auth
+from helpers.auth.error_handler import SaveContextManager
+from utilities.validations import (
+    validate_empty_fields,
+)
+
+
+class Structure(SQLAlchemyObjectType):
+    """Autogenerated Return type for OfficeStructure"""
+
+    class Meta:
+        model = StructureModel
+
+
+class CreateOfficeStructure(graphene.Mutation):
+    """ Returns payload on structure creation"""
```

Comment:
@daniellamarr This docstring could be more descriptive. What does the payload include, is it a message or the structure? Kindly work on this.